service-front/assets/js/moj/moj.cookie-functions.js

```diff
@@ -15,7 +15,7 @@
     cookie_policy: 'essential',
     seen_cookie_message: 'essential',
     _ga: 'usage',
-    _gid: 'usage',
+    _ga_1DVC295G9L: 'usage',
```

Comment:
Does this stay the same all the time?

Reply:
Yep always the same, but this ID is what I'll be changing once this PR passes code review, to the production one in GA